Dockerfile

```diff
@@ -183,7 +183,7 @@ HEALTHCHECK --interval=5m --timeout=3s \
   CMD curl -f http://localhost:${ATLANTIS_PORT:-4141}/healthz || exit 1
 
 # Set up the 'atlantis' user and adjust permissions
-RUN useradd --create-home --user-group --shell /bin/bash atlantis && \
+RUN useradd --uid 1000 --create-home --user-group --shell /bin/bash atlantis && \
```

Comment:
Thanks! I still need to specify the --uid argument, right?

Reply:
When I make this change to the Dockerfile it fails with: `useradd: UID 100 is not unique`, because the user gets created after the `apt-get install` statement runs.

Reply:
It could be made part of the debian_base step:
https://github.com/runatlantis/atlantis/blob/6da77d83a1df4ad7cf7687dc5a802a5e0ee46c88/Dockerfile#L44-L62

Reply:
That would work indeed, I see this is caused by openssh-server that creates two users:

```sh
messagebus:x:100:101::/nonexistent:/usr/sbin/nologin
sshd:x:101:65534::/run/sshd:/usr/sbin/nologin
```